data-prepper-core/src/main/java/org/opensearch/dataprepper/core/pipeline/Pipeline.java

```diff
@@ -236,8 +236,11 @@ private synchronized void startSourceAndProcessors() {
                         }
                     }
             ).collect(Collectors.toList());
+            final ProcessorRegistry processorRegistry = new ProcessorRegistry(processors);
```

Comment:
Why same name as the class member?

Reply:
Good call. I renamed this as well as some other fields to try to add clarity.